man/man8/tc.8

```diff
@@ -443,7 +443,8 @@ see the man pages for individual qdiscs.
 RATES
 Bandwidths or rates.
 These parameters accept a floating point number, possibly followed by
-a unit (both SI and IEC units supported).
+either a unit (both SI and IEC units supported), or a '%' character when spec
+ifying the rate as a percentage of the device's speed.
```

Comment:
Let's make this addition into:

", or a float followed by `%` to specify the rate as a percentage of the device's speed (e.g. 5%, 99.5%)."